test/scheduler_test.js

```diff
@@ -178,5 +179,17 @@ suite('Scheduler', function() {
       assume(new Date(updatedHook.lastFire.time) - new Date()).is.approximately(0, 2000); // 2s slop
       assume(updatedHook.nextScheduledDate).is.not.equal(oldScheduledDate);
     });
+
+    test('on error, notify is used with correct options', async () => {
+      let oldTaskId = hook.nextTaskId;
+      let oldScheduledDate = hook.nextScheduledDate;
+
+      creator.shouldFail = true;
+
+      assume(scheduler.notify.credentials).exists();
+      assume(scheduler.notify.authorizedScopes).exists();
+      assume(scheduler.notify.email).exists();
```

Comment:
These three `assume` options are just assuming that the the fake is set up correctly, rather than checking anything about how the scheduler called it.  I like the use of `assume` in the fake itself -- that's a fine approach.

Another approach would be to record the arguments to the `notify.email`call (say in `notify.lastEmail`), then check them in the test case.  Something like:

```
  assume(scheduler.notify.lastEmail.address).to.equal('someone@somewhere.com');
```